src/vivarium_census_prl_synth_pop/results_processing/formatter.py

```diff
@@ -58,7 +58,12 @@ def format_data_for_mapping(
     obs_results: Dict[str, pd.DataFrame],
     output_columns: List[str],
 ) -> pd.DataFrame:
-    data_to_map = [obs_data[output_columns] for obs_data in obs_results.values()]
+
+    data_to_map = [
+        obs_data[output_columns]
+        for obs_data in obs_results.values()
+        if set(output_columns).issubset(set(obs_data.columns))
+    ]
```

Comment:
This is the same check we do in make results so we do not try to use columns and observer does not have.